yinsolidated/_version.py

```diff
@@ -2,4 +2,4 @@
 
 """Yinsolidated version"""
 
-__version__ = "2.0.3"
+__version__ = "2.0.4"
```

Comment:
_technically_ this seems like a minor-version since it's a "new feature" but I'm not gonna say it needs to block this PR